HISTORY.md

```diff
@@ -1,3 +1,5 @@
+## 3.1.0
```

Comment:
general question - can we apply our established branch naming practice? I wouldn't expect anything prefixed with `test` to be merged down to master, and I believe we have a ticket for this so might be nice to include that in branch name for easy tracking